### PR TITLE
feat(deps): bump deps

### PR DESCRIPTION
**Pull request by odsod**

### CODE REVIEW

File: dialservice.go [hunk 1]
```diff
@@ -28,7 +28,6 @@ func DialService(ctx context.Context, target string, opts ...grpc.DialOption) (*
 					run.requestLoggerMiddleware.GRPCUnaryClientInterceptor,
 					run.clientMiddleware.GRPCUnaryClientInterceptor,
 				),
-				grpc.WithBlock(),
```
Comment (alethenorio) [inline dialservice.go:31]: What is the reason behind this change? Feels like it could have an impact on services which are (intentionally or not) depending on this.

Comment (odsod) [reply dialservice.go:31]: They've deprecated it

grpc: Deprecate WithBlock, WithReturnConnectionError, FailOnNonTempDialError which are ignored by NewClient (https://github.com/grpc/grpc-go/pull/7097)

**Review by alethenorio — APPROVED** [review]
